docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
   server:
     container_name: polis-server
     env_file: ./server/docker-dev.env
-    image: polisdemo/polis-server:dev
+    image: polisdemo/server
```

Comment:
@ballPointPenguin Should we explicitly make these `latest` so it's more greppable, instead of relying on implied defaults?

Reply:
```suggestion
    image: polisdemo/server:latest
```